R/run.R

```diff
@@ -222,7 +223,7 @@ run_one <- function(bm,
   all_params <- list(...)
 
   # separate the global parameters, and make sure only those that are specified remain
-  global_param_names <- c("lib_path", "cpu_count", "mem_alloc")
+  global_param_names <- c("lib_path", "cpu_count", "mem_alloc", "drop_caches")
```

Comment:
Yeah, this kind of use case is one of the biggest for that ticket. It _sounds_ ridiculously liberal, but in cases like this it will help us get out of the trap of "Well we can't update this code cause it'll break history and we need a transition there and and and"